toolbox/cluster/set_scale.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if [ -z "${2-}" ] && [ -z "${3-}" ]; then
+    echo "USAGE: ${0} <instance_type> <scale> [--force]"
+    echo "
+For a list of AWS instance_types see https://aws.amazon.com/ec2/instance-types/ - check the 'Accelerated Computing' section for instances with GPU 
```

Comment:
can you mention explicitly `g4dn.xlarge` as an example of cheap GPU node, and `m4.large` as the default worker node type (correct me if I'm wrong on this one)

Reply:
If unspecified, [the installer chooses between m4.large and m5.large according to availability](https://github.com/openshift/installer/blob/7a6374014536d5993e444e961812dd8adb2ed3fd/pkg/asset/machines/worker.go#L159)